src/analyze/analyze-security.c

```diff
@@ -119,34 +47,34 @@ struct security_assessor {
         bool default_dependencies_only;
 };
 
-static void security_info_free(struct security_info *i) {
+static void security_info_free(security_info *i) {
         if (!i)
                 return;
 
         free(i->id);
-        free(i->type);
-        free(i->load_state);
+        free((char *)i->type);
+        free((char *)i->load_state);
         free(i->fragment_path);
 
         free(i->user);
 
-        free(i->protect_home);
-        free(i->protect_system);
+        free((char *)i->protect_home);
+        free((char *)i->protect_system);
```

Comment:
this is wrong. const strings cannot be freed. If they are freed they cannot be const